locale/zh/messages.json

```diff
@@ -286,13 +300,15 @@
   "Docker Engine": "Docker 引擎",
   "Docker Engine does not support GPU resources, please select Universal Container Runtime (UCR) if you want to use GPU resources.": "Docker 引擎不支持 GPU 资源，如果您想使用 GPU 资源，请选择通用容器运行时 (UCR)。",
   "Docker Runtime only supports the default size for implicit volumes, please select Universal Container Runtime (UCR) if you want to modify the size.": "Docker 运行时仅支持隐式卷的默认大小，如果要修改大小，请选择通用容器运行时 (UCR)。",
+  "Docker parameters": "",
+  "Docker's container runtime. No support for GPU resources and requires image.": "",
   "Docker’s container runtime. No support for multiple containers (Pods) or GPU resources.": "Docker 的容器运行时。不支持多个容器（Pod）或 GPU 资源。",
   "Documentation": "文档",
   "Don't run app tasks on a particular set of attribute IDs": "不要在特定的属性 ID 集合上运行应用程序任务",
   "Download": "下载",
   "Download Config": "下载配置",
   "Download Snapshot": "下载快照",
-  "Download and install: <0><1/> Download dcos.exe</0>.": "下载和安装：<0><1/> 下载 dcos.exe</0>.",
```

Comment:
Deleting this space destroyed the translation 😭 but you could manually put it back